pkg/operation/terraformer/config.go

```diff
@@ -198,9 +198,9 @@ func (t *Terraformer) ConfigExists() (bool, error) {
 	return numberOfExistingResources == numberOfConfigResources, err
 }
 
-// cleanupConfiguration deletes the two ConfigMaps which store the Terraform configuration and state. It also deletes
+// CleanupConfiguration deletes the two ConfigMaps which store the Terraform configuration and state. It also deletes
 // the Secret which stores the Terraform variables.
-func (t *Terraformer) cleanupConfiguration(ctx context.Context) error {
+func (t *Terraformer) CleanupConfiguration(ctx context.Context) error {
```

Comment:
Why is this now public?

Reply:
It's called [here](https://github.com/gardener/gardener/pull/850/files#diff-33d2a8cacfcd2dda84699a11862f3ab0R185) to cleanup the old DNS Terraform files.